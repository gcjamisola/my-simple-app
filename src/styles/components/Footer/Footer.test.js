import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import {StyledFooter} from './index';

test('it works', () => {
	const tree = renderer.create(<StyledFooter/>).toJSON();
	expect(tree).toMatchSnapshot();
});