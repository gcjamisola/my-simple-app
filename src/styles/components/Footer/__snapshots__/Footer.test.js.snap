// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  background: #000000;
  color: #f59500;
  height: 50px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
/>
`;
